tests/acceptance/features/apiReshare/disableReshare.feature

```diff
@@ -0,0 +1,58 @@
+@api @env-config
+Feature: share by disabling re-share
```

Comment:
@saw-jan If I'm not wrong this is a new feature that's noly availabe in `master` and since we have created a new feature file, is it really necessary to backport to stable as well?

Reply:
Yes, it should work. Since the scenarios pass, that means it is working
See that the expected share permission comes down to `15`